### PR TITLE
Fix/check revoked claims proof req

### PR DESCRIPTION
**Pull request by rauljareno**

### CODE REVIEW

File: lib/iden3comm/domain/use_cases/get_filters_from_iden3msg_use_case.dart [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+class GetFiltersFromIden3MsgParam {
+  final Iden3MessageEntity message;
+
+  GetFiltersFromIden3MsgParam({
+    required this.message,
+  });
+}
+
+class GetFiltersFromIden3MsgUseCase
```
Comment (Plumillon) [inline lib/iden3comm/domain/use_cases/get_filters_from_iden3msg_use_case.dart:30]: Same comment about the naming, the specification should be done in Presentation layer.

File: lib/iden3comm/domain/use_cases/get_proofs_from_iden3msg_use_case.dart [hunk 2]
```diff
@@ -4,51 +4,57 @@ import 'package:polygonid_flutter_sdk/iden3comm/domain/entities/iden3_message_en
...
       {required this.message,
       required this.did,
       required this.profileNonce,
       required this.privateKey,
       this.challenge});
 }
 
-class GetProofsUseCase
-    extends FutureUseCase<GetProofsParam, List<JWZProofEntity>> {
+class GetProofsFromIden3MsgUseCase
```
Comment (Plumillon) [inline lib/iden3comm/domain/use_cases/get_proofs_from_iden3msg_use_case.dart:37]: Same comment about naming.

File: lib/sdk/iden3comm.dart [hunk 3]
```diff
@@ -5,18 +5,32 @@ import 'package:polygonid_flutter_sdk/iden3comm/domain/entities/request/auth/aut
...
   /// Returns a [Iden3MessageEntity] from a message string
   Future<Iden3MessageEntity> getIden3Message({required String message});
 
   /// get the vocabulary json-ld files to translate the values of the schemas
   /// to show them to end users in a natural language format in the apps
-  Future<List<Map<String, dynamic>>> getVocabsFromIden3Message(
-      {required Iden3MessageEntity message});
+  /// TODO: remove or fix
+  //Future<List<Map<String, dynamic>>> getVocabs(
+  //    {required Iden3MessageEntity message});
```
Comment (Plumillon) [inline lib/sdk/iden3comm.dart:24]:
```suggestion
```

File: lib/iden3comm/domain/use_cases/get_claims_from_iden3msg_use_case.dart [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+  final String privateKey;
+
+  GetClaimsFromIden3MsgParam(
+      {required this.message,
+      required this.did,
+      required this.profileNonce,
+      required this.privateKey});
+}
+
+class GetClaimsFromIden3MsgUseCase
```
[the remark this answers is not in the document]
Comment (rauljareno) [reply lib/iden3comm/domain/use_cases/get_claims_from_iden3msg_use_case.dart:35]: it's for differenciating from get_claims_use_case, could be better to name it get_iden3comm_claims_use_case?

[the remark this answers is not in the document]
Comment (rauljareno) [reply lib/iden3comm/domain/use_cases/get_claims_from_iden3msg_use_case.dart]: done

**Pull request closed without merging**